varia/style.scss

```diff
@@ -28,11 +28,6 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
  */
 @import "sass/abstracts/imports";
 
-/**
- * Extends
- */
-@import "sass/base/extends";
-
```

Comment:
Another duplicate entry. 
extends is already part of next line `@import "sass/base/imports";`